flexmeasures/api/v3_0/assets.py

```diff
@@ -1107,6 +1107,67 @@ def update_default_asset_view(self, **kwargs):
             "message": "Default asset view updated successfully.",
         }, 200
 
+    @route("/default_legend_position", methods=["POST"])
+    @as_json
+    @use_kwargs(
+        {"default_legend_position": fields.Boolean(required=False)}, location="json"
```

Comment:
The parameter should also change name accordingly to `keep_legends_below_graphs`.